src/global.scss

```diff
@@ -553,15 +553,15 @@ svg.map-svg path {
   fill: none;
 }
 
-.trendline-selected.covid19Vaccination {
+.trendline-selected.covid-19-vaccination {
```

Comment:
Is prettier applying these naming rules?

Reply:
No I'm changing them for consistency.